integration/snyk.io/cucumber/snyk.steps.js

```diff
@@ -0,0 +1,28 @@
+/* eslint
+   new-cap: 0,
+   func-names: 0,
+   prefer-arrow-callback: 0,
+   no-console: 0,
+*/
+const env = require('./environment');
+const Snyk = require('../pages/snyk.io');
+const chai = require('chai');
```

Comment:
chai seems to be not required. 
We can save installing two dependancies if you use the [builtin assert module](https://nodejs.org/docs/latest-v8.x/api/assert.html) instead

Reply:
The only reason to prefer chai is for its output format , from readablity view, during the failure condition. 
Also we are already using it on SSLyze and it proved it to be usefull to us during our demos, we continued to use it here as well.